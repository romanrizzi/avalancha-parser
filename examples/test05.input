
-- Feature específica del parser cuando hay una expresión sola

-- Lo siguiente:
            check f()

-- Debería dar lo mismo que lo siguiente:
            check f() == True

