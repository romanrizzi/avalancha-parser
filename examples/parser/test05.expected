[
   "program",
   [],
   [
      [
         "check",
         [
            "equal",
            [
               "app",
               "f",
               []
            ],
            [
               "cons",
               "True",
               []
            ]
         ]
      ],
      [
         "check",
         [
            "equal",
            [
               "app",
               "f",
               []
            ],
            [
               "cons",
               "True",
               []
            ]
         ]
      ]
   ]
]
